doc/Type/Block.pod6

```diff
@@ -10,11 +10,13 @@ A C<Block> is a code object meant for small-scale code reuse.  A block is
 created syntactically by a list of statements enclosed in curly braces.
 
 Without an explicit signature or placeholder arguments, a block has C<$_>
-as a positional argument
+as a positional argument. Which defaults to the outer scope's $_. Thus it
+will inherit the L<topic|/language/variables#index-entry-topic_variable> if possible.
```

Comment:
Maybe clarify "if possible". You probably mean "unless it's called with an argument".